plone/app/upgrade/v51/betas.py

```diff
@@ -124,15 +123,24 @@ def update_social_media_fields(context):
     field values as they are now declared as ASCIILine instead of
     TextLine.
     """
+    twitter_username = ''
+    facebook_app_id = ''
+    facebook_username = ''
     from Products.CMFPlone.interfaces.controlpanel import ISocialMediaSchema
     registry = getUtility(IRegistry)
     settings = registry.forInterface(ISocialMediaSchema, prefix='plone')
     if settings.twitter_username:
-        settings.twitter_username = str(settings.twitter_username)
+        twitter_username = settings.twitter_username.encode('ascii', 'ignore')
     if settings.facebook_app_id:
-        settings.facebook_app_id = str(settings.facebook_app_id)
+        facebook_app_id = settings.facebook_app_id.encode('ascii', 'ignore')
     if settings.facebook_username:
-        settings.facebook_username = str(settings.facebook_username)
+        facebook_username = settings.facebook_username.encode(
+            'ascii', 'ignore')
+    loadMigrationProfile(context, 'profile-plone.app.upgrade.v51:to51beta4')
```

Comment:
Does this work without losing the data in the config panel?
Because simple loading the registry settings will delete the data of these fields, which I want to avoid here.